client/my-sites/plan-features/style.scss

```diff
@@ -277,10 +277,17 @@ $plan-features-sidebar-width: 272px;
 	}
 }
 
-.plan-features__header-tip-info svg {
-	padding: 0 3px;
-	position: absolute;
-	transform: translateY( -33% );
+.info-popover.plan-features__header-tip-info {
+	position: relative;
+	top: 4px;
+	left: 3px;
+	margin-top: -10px;
+	margin-bottom: -10px;
+	display: inline-block;
+}
+
+.popover.plan-features__header-tip-info .popover__inner {
+	white-space: nowrap;
```

Comment:
The primary change is this one. When the Popover is close to the viewport border and it doesn't have defined the width of its inner box, for some reason the positioning process fails.
The simple fix is set `white-space: nowrap` or directly set a width of the inner box.

```sass
.popover.plan-features__header-tip-info .popover__inner {
  width: 200px;
}
```

I choose the first one because it will work independently of the text inside of the Popover.